sdk/storage/Azure.Storage.Blobs/src/PartitionedDownloader.cs

```diff
@@ -30,20 +31,40 @@ internal class PartitionedDownloader
 
         public PartitionedDownloader(
             BlobBaseClient client,
-            StorageTransferOptions transferOptions = default,
-            long? initialTransferLength = null)
```

Comment:
The PartitionedDownloader will now actually respect StorageTransferOptions.PartitionedDownloader